### ISSUE TITLE
Type Error: fd must be a file descriptor

### ISSUE THREAD

**Issue by hebbianloop** [bids-standard/legacy-validator#450]
Hi, 

Running bids-validator 0.26.2 on a dataset converted with dcm2bids 1.1.8. Directory was initialized with dcm2bids_scaffold then populated with data from 'sourcedata'

Every time I execute ```bids-validator . --verbose``` in this directory, I get the following error

```fs.js:686
  binding.read(fd, buffer, offset, length, position, req);
          ^

TypeError: fd must be a file descriptor
    at Object.fs.read (fs.js:686:11)
    at /usr/local/lib/node_modules/bids-validator/utils/files.js:251:20
    at FSReqWrap.oncomplete (fs.js:153:20)```

**Comment by chrisgorgo** [comment]
My best guess is that there was an issue opening one of the files (wrong permissions? network drive malfunction?). The validator needs to be improved by better handling of such situation here https://github.com/INCF/bids-validator/blob/master/utils/files.js#L250

**Comment by chrisgorgo** [comment]
This was most likely fixed by #484
